### DISCUSSION TITLE
Additional community voted "Difficulty Rating"

### DISCUSSION THREAD

**Discussion by DeityMars** [ppy/osu#26891] [Ideas] [answered]
It's understandably difficult to determine a map's true difficulty with the star rating calculations alone. An example would be a high density low ar map. Although the SR shows something like 4-5* it could feel alot more difficult than that. It could be possible to implement an additional difficulty rating which players could vote for at the results screen that would be shown on the beatmap's page. (similar function to how a player can rate the beatmap at results screens) It could be done in broad tiers like 'simple - intermediate - complex'

**Comment by Rekunan** [comment, marked as answer]
I feel like this feature would just be made obsolete, because it is a part of future pp dev plans to account for reading, and anything else that isn't currently covered, since the main goal for pp is to be fully objective.

**Comment by cihe13375** [comment]
I think this feature has its merit:
1) even though the ultimate goal of pp system is to be "completely" balanced, at the current pace of development I don't foresee it happening in even years;
2) the community voting can be used as a reference guiding the development of the pp system. (a) it can be used as a baseline to calibrate any subjective parts or "free parameters" in pp, which are kinda inevitable (e.g. aim vs. speed, sharp angle vs. large angle, etc.); (b) it can indicate in which direction the pp system should move forward (by e.g. extracting common features from the 5% most overrated map), which is likely better then e.g. looking at a few maps that "happens to" stand out.

However, a major concern is that players would probably not vote faithfully,  e.g. maybe they would vote underrated for the type of maps they like, hoping them get buffed. A probably better approach is using best plays, i.e. if a map has 100k plays but never appears in a user's top 10 or smth then it's likely underrated. That's a bit off-topic though.

**Reply by Rekunan** [reply]
1. I still feel like it would be wasted work if it will be replaced in the future.
2. That sounds cool, but the pp devs already have an idea of what needs to be focused on, and yeah as you said, there are massive issues that come with just pure direct public voting. Perhaps just letting the pp committee vote could be good, but I think they'd rather spend their time pp deving.

**Comment by ghost** [comment]
I don't think this is a recognized idea, even users with the same Rank have different opinions on the difficulty of the same map.
For example, a jump map would be considered very simple by users familiar with jump, while a stream map would be very simple for users familiar with stream.
Allowing users to judge the difficulty is very subjective and unreasonable.
I think it would be more reasonable to divide the difficulty of maps into different technical points, such as aim and speed and more.
The current star rating can only indicate the overall difficulty of the map. Although some maps are unreasonable, the overall trend is correct.
Using certain algorithms to divide the technical difficulties required for a map and attach the difficulty level of each difficulty will make such judgments more meaningful.

**Reply by Rekunan** [reply]
It would technically be possible to show some sort of skill splitting, since pp already has skills and is just summing them geometrically to get the final values.

**Reply by Joehuu** [reply]
https://github.com/ppy/osu/pull/26482 does that (for difficulty) but the preview is kinda barebones right now.
